components/molecule/selectPopover/src/index.js

```diff
@@ -258,11 +278,24 @@ function MoleculeSelectPopover({
     renderContentWrapperProp && `${BASE_CLASS}--hasCustomWrapper`
   )
 
+  const overlayClassNames = cx(
+    `${BASE_CLASS}-overlay`,
+    `${BASE_CLASS}-overlay--${overlayType}`,
+    {'is-open': isOpen}
```

Comment:
not needed

Reply:
if needed, better use an accessibility proper solution like `aria-hidden: false`

Reply:
REMOVED, DONE!